pkg/fanal/artifact/sbom/sbom.go

```diff
@@ -81,7 +81,7 @@ func (a Artifact) Inspect(_ context.Context) (types.ArtifactReference, error) {
 	}
 
 	return types.ArtifactReference{
-		Name:    a.filePath,
+		Name:    filepath.ToSlash(a.filePath),
```

Comment:
this change is here because of `\` and `/` differences between Linux and Windows filesystems. Otherwise tests were failing because of wrong path generated.
Do you think that its change is ok?

Reply:
Test I am udpating `sbom_test.go` is failing without that change, because of Windows filesystem.

<img width="444" alt="image" src="https://github.com/aquasecurity/trivy/assets/457542/13c02f0e-be2f-4978-9bc2-12033124a74f">

Reply:
We run `integration` tests only for linux system - https://github.com/aquasecurity/trivy/blob/74dc5b6804c9285acc86e334add5383ac144725d/.github/workflows/test.yaml#L69